packages/mattermost-redux/src/store/configureStore.ts

```diff
@@ -74,5 +77,8 @@ export default function configureStore<S extends GlobalState>({
         });
     }
 
-    return store;
+    return {
+        store,
+        sagaMiddleware,
+    };
```

Comment:
Similarly, I might need to figure out a different way to give outside code access to the `sagaMiddleware`. It turns out that there's a log of testing code that calls `configureStore` directory so this is breaking a bunch of places.